image/templates/helm/shared/templates/02-scanner-06-deployment.yaml.htpl

```diff
@@ -49,6 +49,31 @@ spec:
       {{- if ._rox.scanner.priorityClassName }}
       priorityClassName: {{ ._rox.scanner.priorityClassName }}
       {{- end }}
+      {{- if ._rox._securedClusterCertRefresh }}
+      initContainers:
+        - name: init-tls-certs
```

Comment:
Here and in most other files the indentation of `initContainers` list seems incorrect. I think it should follow the pattern of `image/templates/helm/shared/templates/02-scanner-v4-07-db-deployment.yaml`.